rpc/params/crossmodel.go

```diff
@@ -76,11 +76,16 @@ type ApplicationOfferDetails struct {
 	OfferName              string             `json:"offer-name"`
 	ApplicationDescription string             `json:"application-description"`
 	Endpoints              []RemoteEndpoint   `json:"endpoints,omitempty"`
-	Spaces                 []RemoteSpace      `json:"spaces,omitempty"`
-	Bindings               map[string]string  `json:"bindings,omitempty"`
 	Users                  []OfferUserDetails `json:"users,omitempty"`
 }
 
+// ApplicationOfferDetails represents an application offering from an external model.
+type ApplicationOfferDetailsV4 struct {
+	ApplicationOfferDetails
```

Comment:
We are discussing whether to rename ApplicationOfferDetails to ApplicationOfferDetailsV5 etc

Leaning towards just leaving it as ApplicationOfferDetails. Just need to confirm.

Reply:
done, now the api and structs contain explicitly the latest version